src/main/resources/sql/schema/schema-tables.sql

```diff
@@ -120,3 +120,8 @@ CREATE TABLE citation_datasource_municipality (
   citation_datasource_id    INTEGER,
   municipality_id           INTEGER
 );
+
+CREATE TABLE tyler_court_mapping(
+  court_id				INTEGER			NOT NULL,
+  datasource_identifier VARCHAR(25)

```

Comment:
Since this table is specific to tyler, maybe use tyler_court_identifier instead of datasource_identifier. If this table was generic, I would agree. But since it's not, the terminology can confuse since it may seem like it's the actual id of the datasource.